recipes/libcint/0001-Fix-test-location.patch

```diff
@@ -0,0 +1,38 @@
+From 6d92c5c2ef0a2e4856bf79fc22dd00f5bf34f5a9 Mon Sep 17 00:00:00 2001
+From: BastianZim <10774221+BastianZim@users.noreply.github.com>
+Date: Wed, 3 Nov 2021 15:27:31 +0100
+Subject: [PATCH] Fix test location
+
+---
+ testsuite/test_3c2e.py | 2 +-
+ testsuite/test_cint.py | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/testsuite/test_3c2e.py b/testsuite/test_3c2e.py
+index 70bbad3..7715116 100644
+--- a/testsuite/test_3c2e.py
++++ b/testsuite/test_3c2e.py
+@@ -13,7 +13,7 @@ import os
+ import ctypes
+ import numpy
+
+-_cint = numpy.ctypeslib.load_library('libcint', os.path.abspath(os.path.join(__file__, '../../build')))
++_cint = numpy.ctypeslib.load_library('libcint', os.path.join(os.environ.get('PREFIX'), 'lib'))
```

Comment:
just fyi a better way is probably to use `import sys; sys.prefix`. But since this is only the tests, probably fine!

Reply:
Ahh TIL. To be honest I just copied your approach from [or-tools](https://github.com/conda-forge/or-tools-feedstock/blob/1937ab734ad7e9b127e7734efccef32a7961b2ca/recipe/meta.yaml#L57) because I was slowly going insane finding a solution...

Reply:
yeah the $PREFIX environment variable is _not_ available at runtime (only in the tests).

At runtime we (should) have `$CONDA_PREFIX`, but it's _only_ there when ran with `conda activate ...` before.

So best to rely on the "fixed" prefix in the python interpreter (which works even without `conda activate`.

Reply:
Ahh, now it's more clear, thanks! :)

Reply:
Leet me change it to sys then just for best practices.